wenet/cif/embedding.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2023 ASLP@NWPU (authors: He Wang, Fan Yu)
```

Comment:
The embedding.py shares almost the same code with transformer/embedding.py, I think we can reuse it.